compute/server-groups.rst

```diff
@@ -0,0 +1,52 @@
+=============
+Server groups
+=============
+
+By creating groups of instances, its possible to assign various functions from the platform onto the members of the group. The most common feature is "affinity" (or "anti affinity") wherein you can force instances to be on the same underlying hardware ("hypervisor") - or vice versa (to force instances onto separate hypervisors). 
+
+Affinity types
+--------------
+In Binero Cloud, we support four versions of affinity:
+
+- **affinity** will force instances in the same group to be hosted on the same hardware. It is useful to achieve the quickest possible network speed between them (traffic will never enter the networking but remain in the hypervisor CPU) but will bring down all instances in the group in the event of the hypervisor crashing.
+- **anti-affinity** does the reverse of affinity, it will provision instance on different hypervisors and therefore guarantee that two grouped instances will never be affected by a single hypervisor crash. 
+- **soft-affinity** does the same as affinity but will only attempt to place on same hardware. If it fails for some reason (the hypervisor is oversubscribed or not available) the provisioning finds an alternative hypervisor according to best effort.
+- **soft-anti-affinity** does the same as anti-affinity but will only attempt to place on different hypervisors. If it fails for some reason (there is not enough hypervisors available for the type of resource, for instance) the provisioning finds an alternative hypervisor according to best effort.
+
+.. Important::
+	We **strongly** recommend using the "soft" versions. This is because we from time to time move instance around in the platform during maintenance and if an instance cannot meet the requirement (due to affinity), it will be powered off. The soft version will always be able to be migrated.
+
+Anti affinity
```

Comment:
Anti-affinity, we should keep it consistent in this entire document with ``anti-affinity`` instead of ``anti affinity``